src/components/editor/editor-pane/one-dark.scss

```diff
@@ -1,13 +1,10 @@
-/**
- * Atom One Dark Theme
- *
+/*
  * Copyright (c) 2015 Hikio - twitter.com/hik_io
+ * SPDX-FileCopyrightText: 2015 Hikio - twitter.com/hik_io
  *
- * 06/26/2015
- *
- * Licensed under MIT
- * GitHub https://github.com/hikio/brackets-one-dark
```

Comment:
Maybe it's useful to keep the GitHub repo and the theme name somewhere around.

Reply:
fixed